src/index.js

```diff
@@ -54,88 +50,20 @@ const showPostsByTimeline = (timeline) => {
 }
 
 /**
- * This is a function to display name and description of author.
- * @param {Object} profile - profile of the author
- * @returns {null} null
- */
-
-const showNameAndDesc = (profile) => {
-    log(title('name'))
-    log(body(profile.name));
-    log(title('\ndescription'))
-    log(body(`${profile.desc}\n`));
-}
-
-/**
- * This is a function to display author profile details such as email,work,location,etc.
- * @param {Object} profile - profile of the author
+ * This is a function to display the profile details of the author.
+ * @param {Object} profileInfo - profile of the author
```

Comment:
I think this is wrong please check.

Reply:
Sorry, my bad.
Also one doubt, why are mentioning all this in comments. Is this only for coders to understand or is it some kind of automatic documentation?

Reply:
Well, It has many benefits but most obvious is of course so that other developer can understand but this also helps in Intellisense of VSCode. Like this:
![](https://media.giphy.com/media/KX7FcaEzVScqplqy16/giphy.gif)

Reply:
Oh great, I didn't know of this. Thanks.